net-dns/knot/files/2.0.0_beta-spell-enable-vars-correctly.patch

```diff
@@ -1,11 +0,0 @@
---- configure.orig	2015-05-10 19:25:43.365929384 +0200
-+++ configure	2015-05-10 19:26:00.452648532 +0200
-@@ -12833,7 +12833,7 @@
- if test "$enable_daemon" = "yes"; then :
- 
- 
--if test "$enable_system" != "no"; then :
-+if test "$enable_systemd" != "no"; then :
```

Comment:
This typo has been fixed since 2.0.0-beta, so this entire patch is no longer needed.